include/entity.h

```diff
@@ -671,6 +671,23 @@ typedef struct {
     s16 unk8A;
 } ET_BatEcho;
 
+typedef struct {
+    /* 0x7C */ s16 pad7C;
+    /* 0x7E */ s16 unk7E;
+    /* 0x80 */ s16 unk80;
+    /* 0x82 */ s16 unk82;
+    /* 0x84 */ s16 unk84;
+    /* 0x86 */ s16 unk86;
+    /* 0x88 */ s16 unk88;
+    /* 0x8A */ s16 unk8A;
+    /* 0x8C */ s16 unk8C;
+    /* 0x8E */ s32 pad8E;
+    /* 0x92 */ struct Entity* unk92;
```

Comment:
Not critical right now but these pointers are going to be out of alignment if compiled in 64-bit mode

Reply:
I'd like to assume by that point, we probably won't need to have these named as such (or the comment with where they are in the struct).  I can remove the comments if you'd like, but I don't really know what else to name the variables.

Reply:
I don't mean the comments, I'm not sure exactly what the issue will be but there's most likely going to be portability bugs when the time comes. Here's examples of static asserts we use to check alignment on structs https://github.com/Xeeynamo/sotn-decomp/blob/fb8e5d795083256425400782835e6e7de8ef1442/include/entity.h#L1958

You don't need to change this to get it merged, I just thought it should be pointed out

Reply:
Ok.  I'll look into setting that up for this in another PR (as well as setting that up for ET_Bat as well).